common/runtime/model.go

```diff
@@ -158,63 +158,178 @@ type OIDCConfigDTO struct {
 	SigningAlgs    []string `json:"signingAlgs" yaml:"signingAlgs"`
 	UsernameClaim  string   `json:"usernameClaim" yaml:"usernameClaim"`
 	UsernamePrefix string   `json:"usernamePrefix" yaml:"usernamePrefix"`
+	RequiredClaims []string `json:"requiredClaims" yaml:"requiredClaims"`
 }
 
 const oidcValidSigningAlgs = "RS256,RS384,RS512,ES256,ES384,ES512,PS256,PS384,PS512"
 
-func (o *OIDCConfigDTO) IsProvided() bool {
+func (o *OIDCsDTO) IsProvided() bool {
```

Comment:
Easy to misspell. But I do not get any proposal.

Reply:
To avoid so many upper case letters: `OidcDTO` or this previous name `OIDCConfigDTO` or `OIDConnectDTO`

Reply:
I think that `OIDConnectDTO` is the way. `OIDCConfigDTO` is already in use and thats why a new name was needed.